docs/contacting-support/alcf-users-slack.md

```diff
@@ -0,0 +1,22 @@
+# ALCF Users Slack Workspace
+
+The ALCF Users Slack workspace is a platform intended for current, active ALCF users* where the user community can interact, collaborate, and help one another. This workspace is a pilot program valid for 1 year. 
```

Comment:
is that asterisk a stray?

Reply:
Good catch! The * is expanded in Getting access section. It should be repeated in this section as well.